priv/repo/seeds.exs

```diff
@@ -133,3 +133,15 @@ CoursePlanner.Repo.delete_all CoursePlanner.User
       required: false
     })
 |> CoursePlanner.Repo.insert!
+
+%CoursePlanner.SystemVariable{}
+|> CoursePlanner.SystemVariable.changeset(
+    %{
+      key: "DISABLE_NOTIFICATION",
+      value: "true",
+      type: "boolean",
+      editable: true,
+      visible: true,
+      required: false
```

Comment:
Isn't it enough to call `Settings.get_value("DISABLE_NOTIFICATION", false)` with the default value?